boards/pjrc/teensy4/teensyMM.yaml

```
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2020, Bernhard Kraemer
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: teensyMM
```

```suggestion
identifier: teensymm
```
and filename to lowercase